packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -107,47 +95,47 @@ When a specific client/IP/device identifier combination exceeds either the 60 re
 
 > **Note:** For all orgs created after September 2020, this feature is automatically set to the **Enforce and log per client (recommended)** mode.
 
-To configure client-based rate limiting for existing orgs:
+To configure the client-based rate limit for existing orgs:
 
-1. From the Admin Console select **Settings** > **Account**, and then scroll down to the **Client-based rate limiting** section.
+1. From the Admin Console select **Settings** > **Account**, and then scroll down to the **Client-based rate limit** section.
 
 2. Select the type of **Rate limit per client** that you want to implement:
 
-    * Select **Log and enforce client (recommended)** to enable client-based rate limiting.
+    * Select **Enforce and log per client (recommended)** to enable client-based rate limit.
 
-    * Select **Log per client** to enable client-based rate limiting in preview mode. In **Log per client** mode, rate limiting is based on the org-wide rate-limit values, but client-specific rate limiting error information is logged as System Log events. By analyzing these System Log events, you can determine if client-based rate limiting is effective for you.
+    * Select **Log per client** to enable client-based rate limit in preview mode. In **Log per client** mode, rate limit is based on the org-wide rate limit values, but client-specific rate limit error information is logged as System Log events. By analyzing these System Log events, you can determine if client-based rate limit is effective for you.
 
-    * Select **No action** to disable client-based rate limiting.
+    * Select **No action** to disable client-based rate limit.
```

Comment:
"disable client-based rate limit"
-> "disable the client-based rate limit"
or -> "disable client-based rate limiting"

Reply:
Fixed.